### PR TITLE
propose feed items rewordk

### PR DESCRIPTION
**Pull request by dholms**
Doing this as a separte PR cause I got a little bit carried away but wanted to get feedback 😛

### CODE REVIEW

File: lexicons/app/bsky/feed/getPostThread.json [hunk 1]
```diff
@@ -29,32 +29,17 @@
         {"name": "NotFound"}
       ]
     },
-    "post": {
+    "postInThread": {
       "type": "object",
       "required": ["uri", "cid", "author", "record", "replyCount", "repostCount", "upvoteCount", "downvoteCount", "indexedAt"],
```
Comment (pfrazee) [inline lexicons/app/bsky/feed/getPostThread.json:34]: Dont forget to update these biddies

File: lexicons/app/bsky/feed/viewItem.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.postInFeed",
```
Comment (pfrazee) [inline lexicons/app/bsky/feed/viewItem.json:3]: Bikeshed! I feel like `feedViewPost` is a little less clunky?

Comment (pfrazee) [reply lexicons/app/bsky/feed/viewItem.json:3]: Likewise: `threadViewPost`

File: lexicons/app/bsky/feed/getPostThread.json (hunk 1 above)
Comment (dholms) [reply lexicons/app/bsky/feed/getPostThread.json:34]: oh nice good catch 👍

File: lexicons/app/bsky/feed/viewItem.json (hunk 2 above)
Comment (dholms) [reply lexicons/app/bsky/feed/viewItem.json:3]: yup i like it 👍

**Pull request merged**